doc/Basic-configuration.md

```diff
@@ -27,13 +27,13 @@ There are 2 types of options: params and features. Unlike params, features can b
     * **Example:** `"{host_config, \"localhost2\", [{auth_method, anonymous}, {allow_multiple_connections, false}]}." `
 
 * **pool** - feature
-    * **Description:** A named pool of connections to an SQL DB. To enable the connection, remove '%%' prefix from value and configure it with the `odbc_server` option (see below).
-    * **Syntax:** `"{pool, odbc, PoolName}."` or `"{pool, odbc, PoolName, Options}."`
-    * **Examples:** `"{pool, odbc, default}."`
+    * **Description:** A named pool of connections to an SQL DB. To enable the connection, remove '%%' prefix from value and configure it with the `rdbms_server` option (see below).
```

Comment:
**Description:** A named pool of connections to an SQL DB. To enable the connection, remove '%%' prefix from the value and configure it with the `rdbms_server` option (see below).